src/components/daily/foodtooltip/FoodTooltip.js

```diff
@@ -29,18 +29,45 @@ class FoodTooltip extends React.Component {
     return _.get(food, 'nutrition.carbohydrate.net', 0);
   }
 
+  getPrescribedCarbs(food) {
+    return _.get(food, 'prescribedNutrition.carbohydrate.net', 0);
+  }
+
+  isPrescribed(food) {
+    const prescriptor = _.get(food, 'prescriptor');
+    if (prescriptor !== undefined) {
+      return true;
+    }
+    return false;
+  }
+
   renderFood() {
     const food = this.props.food;
-    const rows = [
+    const actualValue = this.getCarbs(food); 
+    const rows = [];
+    if (this.isPrescribed(food)) {
+      const prescribedValue = this.getPrescribedCarbs(food);
+      rows.push(      
+        <div key={'prescribed'} className={styles.prescribed}>
+        <div className={styles.label}>{i18next.t('Recommended')}</div>
+        <div className={styles.value}>
+          {`${prescribedValue}`}
```

Comment:
Display the units ?

If no, I think it safe to write:
```suggestion
          {prescribedValue}
```
or to be more strict:
```suggestion
          {prescribedValue.toString(10)}
```